tests/shared/src/test/scala/scala/meta/tests/parsers/VarargParameterSuite.scala

```diff
@@ -126,7 +130,7 @@ class VarargParameterSuite extends ParseSuite {
     val error = intercept[parsers.ParseException] {
       templStat(definition)
     }
-    assert(error.getMessage.contains(expected))
+    assert(clue(error).getMessage.contains(clue(expected)))
```

Comment:
We shouldn't need to change this if it was working before.

Reply:
reverted